apps/web/src/pages/studio-onboarding/success.tsx

```diff
@@ -65,6 +66,10 @@ export const StudioOnboardingSuccess = () => {
 
   useEffect(() => {
     segment.track('Test workflow step completed - [Onboarding - Signup]');
+
+    // Never expires! Well it does, in 10 years but you will change device or browser by then :)
+    novuOnboardedCookie.set('1', { expires: 10 * 365 });
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Prefer extracting to a constant.

However, saw this [online](https://stackoverflow.com/a/3290474): 

> As in 2023, obeying the max depends on the web browsers. As of Chrome release M104 (August 2022) cookies can no longer set an expiration date of more than 400 days in the future.‏